src/components/Sidebar/CollectionToolbar.svelte

```
@@ -1,7 +1,7 @@
 <script lang="ts">
 	import * as _userIconCollection from '../../models/UserIconCollection';
-	import { UserIconCollectionService } from '../../services/user-icon-collection.service';
-	import * as _selectedIcon from '../../models/SelectedIcon';
+	import { UserIconCollectionIndexedDBService } from '../../services/user-icon-collection-indexeddb.service';
+	import * as _selectedIcon from '../../models/CustomizableIcon';
```

ESLint

[error] '_selectedIcon' is defined but never used.

(@typescript-eslint/no-unused-vars)